modules/async-icq/testing/demo-simapp/x/interquery/client/cli/tx.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	// "github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/quasar-finance/interchain-query-demo/x/interquery/types"
```

Comment:
should be updated to reflect the current repo vs. quasars repo. could also remove the commented out import